Week 1/Day 3/recipes/css/styles.css

```diff
@@ -0,0 +1,68 @@
+.apple-recipe-header{
+    background-image: url(../images/apple-pie.jpg);
+    background-size:cover ;
+    height:650px;
```

Comment:
I might use a percentage for the height for this so that the page can be viewed on a smaller window.  But we will go over this type of stuff in the next week or two.